cmd/gateway/commands_test.go

```diff
@@ -318,54 +323,76 @@ func TestStaticModeCmdFlagValidation(t *testing.T) {
 		{
 			name: "usage-report-secret is invalid",
 			args: []string{
-				"--usage-report-secret=my-secret", // no namespace
+				"--usage-report-secret=!@#$",
 			},
-			wantErr: true,
-			expectedErrPrefix: `invalid argument "my-secret" for "--usage-report-secret" flag: invalid format; ` +
-				"must be NAMESPACE/NAME",
+			wantErr:           true,
+			expectedErrPrefix: `invalid argument "!@#$" for "--usage-report-secret" flag: invalid format: `,
 		},
 		{
-			name: "usage-report-server-url is set to empty string",
+			name: "usage-report-endpoint is set to empty string",
 			args: []string{
-				"--usage-report-server-url=",
+				"--usage-report-endpoint=",
 			},
 			wantErr:           true,
-			expectedErrPrefix: `invalid argument "" for "--usage-report-server-url" flag: must be set`,
+			expectedErrPrefix: `invalid argument "" for "--usage-report-endpoint" flag: must be set`,
+		},
+		{
+			name: "usage-report-endpoint is an invalid endpoint",
+			args: []string{
+				"--usage-report-endpoint=$*(invalid)",
```

Comment:
nit: should we test this with something like `host::port` or port in the wrong range?

Reply:
We test those types of cases in the `validation_test` file.